homeassistant/components/history.py

```diff
@@ -185,8 +216,77 @@ def get(self, request, datetime=None):
         end_time = start_time + one_day
         entity_id = request.args.get('filter_entity_id')
 
-        return self.json(
-            get_significant_states(start_time, end_time, entity_id).values())
+        return self.json(get_significant_states(
+            start_time, end_time, entity_id, Filters(self.config)).values())
```

Comment:
Let's cache the `Filters` object. If you create the filters in `setup` you don't even have to store the `config`.